app/src/main/java/org/metabrainz/android/presentation/features/search/SearchPagingSource.kt

```diff
@@ -19,18 +19,20 @@ class SearchPagingSource(val mbEntityType: MBEntityType, val query: String) : Pa
         return try {
             val response = service.searchEntity(mbEntityType.entity, query, pageSize, offset)?.string()
             var count = LoadResult.Page.COUNT_UNDEFINED
+            val artistList = JsonParser.parseString(response).asJsonObject.get("artists").asJsonArray
```

Comment:
Artists aren't the only entities that can be searched. What about releases, labels, events, etc?